### PR TITLE
City and postcode should be required fields for event registration

### PR DESCRIPTION
**Pull request by sukhada**
https://github.com/350org/action-tools/issues/258

### CODE REVIEW

File: event_create_field_set.html [hunk 1]
```diff
@@ -21,24 +21,27 @@ <h3 class="section-header meta margin-bottom-large">{% filter ak_text:"event_cre
       		{% include "./country_select.html" %}
       	</div>
         <div class="input-text ak-input-type-event input-address">
-      		{% if page.custom_fields.events_custom_address_field %}{{page.custom_fields.events_custom_address_field}}
-{% else %}
-{% filter ak_text:"field_address1"|capfirst %}Street address{% endfilter %}
-{% endif %}	
+			{% if page.custom_fields.events_custom_address_field %}
+			<label for="id_event_address1">{{page.custom_fields.events_custom_address_field}}<span class="ak-required-flag">*</span></label>
```
Comment (cjmabry) [inline event_create_field_set.html:25]: Is this actually making them required or just adding help text that they are required?

**Review by cjmabry — COMMENTED** [review]
Question below.

File: event_create_field_set.html (hunk 1 above)
Comment (sukhada) [reply event_create_field_set.html:25]: @cjmabry they're already required, this is just adding the appropriate help text to make sure they're marked as required, as well as the "required" label next to them when form validation fails.

Comment (cjmabry) [reply event_create_field_set.html:25]: Got it, thanks!

**Review by cjmabry — APPROVED** [review]

**Pull request merged**